content/en/opentelemetry/collector_exporter/otlp_receiver.md

```diff
@@ -0,0 +1,191 @@
+---
+title: OTLP Receiver
+further_reading:
+- link: "/opentelemetry/collector_exporter/"
+  tag: "Documentation"
+  text: "Setting Up the OpenTelemetry Collector"
+---
+
+## Overview
+
+To collect OTLP metrics, logs, and traces, configure the [OTLP receiver][1] in your Datadog Exporter.
+
+For more information, see the OpenTelemetry project documentation for the [OTLP receiver][1].
+
+## Setup
+
+Add the following lines to your Datadog Exporter configuration:
+
+```yaml
+receivers:
+  otlp:
+   protocols:
+      http:
+       endpoint: "localhost:4318"
+      grpc:
+       endpoint: "localhost:4317"
+```
+
+## Data collected
+
+Traces, metrics, and logs.
+
+## Full example configuration
+
+For a full working example configuration for the Datadog Exporter, see [`otlp.yaml`][2].
```

Comment:
reword : `for the Datadog Exporter` to  `with the Datadog exporter` in all the pages.